### ISSUE TITLE
Document failed to load

### ISSUE THREAD

**Issue by NecroSith** [avocode/avocode#2189]
---
* **Version**: 3.5.5
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 321906
---
```
Error: Octopus not available
    at e.<anonymous> (file:///C:/Users/NecroSith/AppData/Local/avocode/app-3.5.5/resources/app.asar/dist/avocode.min.js:1:11922864)
    at tryCatch (C:\Users\NecroSith\AppData\Local\avocode\app-3.5.5\resources\app.asar\node_modules\regenerator-runtime\runtime.js:64:40)
    at Generator.invoke [as _invoke] (C:\Users\NecroSith\AppData\Local\avocode\app-3.5.5\resources\app.asar\node_modules\regenerator-runtime\runtime.js:355:22)
    at Generator.prototype.(anonymous function) [as next] (C:\Users\NecroSith\AppData\Local\avocode\app-3.5.5\resources\app.asar\node_modules\regenerator-runtime\runtime.js:116:21)
    at tryCatch (C:\Users\NecroSith\AppData\Local\avocode\app-3.5.5\resources\app.asar\node_modules\regenerator-runtime\runtime.js:64:40)
    at Generator.invoke [as _invoke] (C:\Users\NecroSith\AppData\Local\avocode\app-3.5.5\resources\app.asar\node_modules\regenerator-runtime\runtime.js:297:24)
    at Generator.prototype.(anonymous function) [as next] (C:\
```

**Comment by NecroSith** [comment]
This error appears whenever I try to open any existing design in the desktop app

**Comment by crashmajstr** [comment]
Hi @NecroSith I've just tried to reproduce this behaviour on my end but without any luck... Please, can you send me a screencast of the buggy scenario? 
Thanks, 
Josef

**Comment by NecroSith** [comment]
Hi @crashmajstr 
I investigated this thing a bit further and found out that this error pops up with not opening ANY existing designs but with only some.  They surely could be opened earlier because I worked with em some time ago but now the error occurs every time I try to load them. While loading the design, it kinda stucks on "Loading metadata" and either loads the design or [fails](https://1.downloader.disk.yandex.ru/disk/a820b4ab7c99c00729c3bd31f0a2e435669a612ffa416adde1f5f58d0613e101/5bc49cec/9R9d4mqsqRBxBGO__OEFB2awEAoHsIIsCBo1SosU0_P9yZcghAlUvJ4gZY1Jer5bE6MCVJjChGqvbOrtVznA-w%3D%3D?uid=74248107&filename=2018-10-15_12-50-58.png&disposition=inline&hash=&limit=0&content_type=image%2Fpng&fsize=86171&hid=bab0ff6b10da2f459dfee2eff0b1c559&media_type=image&tknv=v2&etag=17de5c598d0acc224216ec8b137ae4b8)

**Comment by crashmajstr** [comment]
Thanks for the info! I've just tried to reproduce it and I've experienced the same error. I'll hand it out to our dev team for further investigation. For now, please, upload the file again, it worked for me. 
I'll let you if there is any update.
Best,
Josef

**Comment by crashmajstr** [comment]
@NecroSith Hey, It looks like the error I reproduced was a different one because I can open your file just fine. Please, can you try to reupload it and try to open it again? Thanks

**Comment by NecroSith** [comment]
@crashmajstr Hi
I was trying to reupload this psd I couldn't open earlier but now it doesn't seem to upload at all :O
I drag-and-drop it but nothing happens, the same with manual choosing. Jpeg was uploaded and could be viewed normally but not psd

**Comment by crashmajstr** [comment]
@NecroSith oh that's strange. Do you have the latest app? (3.5.6) Also, please, can you try if it's the same in the web app? Thanks

**Comment by NecroSith** [comment]
@crashmajstr yes, I have the lastest version.
I tried to reupload the psd one more time. This time it succeeded but it threw an error while opening the design. Here is the text, I'm not sure if I should post a new issue about this or not.
The web version works fine.






---
* **Version**: 3.5.6
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 321906
---
```
Error: Remote octopus link missing
    at e.value (file:///C:/Users/NecroSith/AppData/Local/avocode/app-3.5.6/resources/app.asar/dist/avocode.min.js:1:3073731)
    at e.<anonymous> (file:///C:/Users/NecroSith/AppData/Local/avocode/app-3.5.6/resources/app.asar/dist/avocode.min.js:1:3072738)
    at tryCatch (C:\Users\NecroSith\AppData\Local\avocode\app-3.5.6\resources\app.asar\node_modules\regenerator-runtime\runtime.js:64:40)
    at Generator.invoke [as _invoke] (C:\Users\NecroSith\AppData\Local\avocode\app-3.5.6\resources\app.asar\node_modules\regenerator-runtime\runtime.js:355:22)
    at Generator.prototype.(anonymous function) [as next] (C:\Users\NecroSith\AppData\Local\avocode\app-3.5.6\resources\app.asar\node_modules\regenerator-runtime\runtime.js:116:21)
    at n (file:///C:/Users/NecroSith/AppData/Local/avocode/app-3.5.6/resources/app.asar/dist/avocode.min.js:1:3070194)
    at file:///C:/Users/NecroSith/AppData/Local/avocode/app-3.5.6/resources/app.asar/dist/avocode.min.js:1:3070
```

**Comment by crashmajstr** [comment]
@NecroSith thanks for the provided info. One thing you can try is to logout/login in the desktop app and see if that helps. If not you can help us by sending me a screencast with devtools enabled (network tab). 
How to enable devtools: click on the Help menu while holding "Alt" key > Troubleshooting > Devtools 

Otherwise, just use the web app if it's working ok.

**Comment by crashmajstr** [comment]
Hi, I was wondering if you get any further regarding this issue.

**Comment by NecroSith** [comment]
@crashmajstr Hi there
Sorry for the late response - had my hands full until recently.
I made a screencast with pretty much everythin happening with the desktop app while working. Check it out if you can open the link. 
https://yadi.sk/i/ZQ8vpwjMq_h38A
Please tell me if you need some more info about anything

**Comment by crashmajstr** [comment]
@NecroSith thank you for sharing that with us. I'll hand it out to our dev for further investigation and let you asap.

**Comment by crashmajstr** [comment]
@NecroSith we're still trying to investigate this issue, but I have a question, do you use any type of antivirus/firewall systems on your PC? 
If so please, can you try to disable it and try the desktop app again?
Let me know, thank you.

**Comment by crashmajstr** [comment]
Hello @NecroSith 
please, I have another question, do you use any proxy? Because it looks like that the desktop app can't reach out to our servers.
However, we've just made a build which may help to solve your issue 
LINK: https://we.tl/t-xZYSjhdJYF
Please, give it a try and let me know if it helped or not. 
Thank you and sorry for the inconvenience.

**Comment by NecroSith** [comment]
@crashmajstr Hello
I don't use any kind of antivirus and firewall on my PC. Also there is no proxy that I'm aware of.
I'm downloading the fix now. I'll let you know about the results

**Comment by NecroSith** [comment]
@crashmajstr 
I've tested the build you provided me with and everything works fine, no problems at all. Dev console still throws at me tons of errors though but I can painlessly create new project and manage the designs. 
The problem seems to be solved thanks to this new build. Send my thanks to the dev team and thank you, you guys are awesome! 
I attached a [screenshot](https://yadi.sk/i/WlE53THjMS7p3g) of the dev console with the mistakes I get. They are pretty much the same as earlier but maybe you will find it useful. I will let you know if there are any errors or malfunctions again. Also tell me if I can help with anything

**Comment by crashmajstr** [comment]
Thanks for letting us know @NecroSith ! Hope everything will be ok from now on, we'll let you know once this change will be in a regular build so please, for now, use this build.

**Comment by crashmajstr** [comment]
Hello @NecroSith I am sorry but we won't be releasing the fix of the problem you had today since it requires more testing. So, use the build I provide you with till the next release, please. 
Also I have a question regarding the problem, do you use VMWare or VirtualBox?
Let me know and sorry for the inconvenience.
Best,
Josef

**Comment by NecroSith** [comment]
Hi @crashmajstr 
Don't worry - take your time guys to make the best possible experience, I'm fine with the build you provided.
Yes, I actually have both VMWare and VirtualBox installed on my PC. I use them from time to time but Avocode isn't installed on any of them.

**Comment by crashmajstr** [comment]
Hello @NecroSith we've just released a new version of Avocode 3.6.10 which contains the new Electron 4 which should have fixed the issue you had. Please give it a try and let me know!
Best,
Josef

**Comment by NecroSith** [comment]
Hi there @crashmajstr 
I have been testing new version of Avocode for some time and It seems this error no longer appears. The problem seems to be solved.
